### PR TITLE
Calling uplink.

### PR DESCRIPTION
**Pull request by TruePaIntGit**
## About the PR
 Added the ability to call from an uplink like a bananaphone.

## Why / Balance
 This will allow rp to solve some of the operatives problems.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
- tweak: From uplink, it is now possible to write to the syndicate.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Please stop making new prs.

**Comment by TruePaIntGit** [comment]
> Please stop making new prs.

I made a mistake in those and shut them down. Sorry, this is my first pr.

**Comment by lzk228** [comment]
i think better would be add a sindyphone or something

File: Resources/Prototypes/Entities/Objects/Specific/syndicate.yml [hunk 1]
```diff
@@ -77,6 +77,11 @@
     preset: StorePresetUplink
     balance:
       Telecrystal: 0
+  - type: Prayable
+    sentMessage: prayer-popup-notify-synducate-sent
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Specific/syndicate.yml:81]:
```suggestion
    sentMessage: prayer-popup-notify-syndicate-sent
```
you should add a locale string, with message what would popup

**Review by lzk228 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/syndicate.yml (hunk 1 above)
Comment (TruePaIntGit) [reply Resources/Prototypes/Entities/Objects/Specific/syndicate.yml]: Should I add the ability for only the aplink owner to use the call?

File: Resources/Prototypes/Entities/Objects/Specific/syndicate.yml [hunk 2]
```diff
@@ -77,6 +77,11 @@
     preset: StorePresetUplink
     balance:
       Telecrystal: 0
+  - type: Prayable
+    sentMessage: You sent a message to the bosses
+    notifiactionPrefix: SYNDICATE
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Objects/Specific/syndicate.yml:82]: No, make a fluent string

**Review by ficcialfaint — CHANGES_REQUESTED** [review]
It's Uplink not Aplink. You forgot a \- in the changelog, see https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog

**Review by AJCM-git — CHANGES_REQUESTED** [review]
A better implementation would be making emmaging a bananaphone/redphone send a message to the syndicate

**Comment by deltanedas** [comment]
webedit ops

**Comment by AJCM-git** [comment]
@TruePaIntGit You coming back to this?

**Comment by Partmedia** [comment]
Please re-open if you'd like to come back to this.

**Pull request closed without merging**